docs/reference/FAQ.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: FAQ
+parent: Reference
+nav_order: 4
+permalink: /reference/faq/
+---
+
+- TOC
+{:toc}
+
+# What languages does ClusterFuzz support?
+
+ClusterFuzz definitely supports C, C++ when compiled with
+[clang](https://clang.llvm.org/). It also supports Rust and it probably supports
```

Comment:
Nit: Instead of explicitly mentioning support in the second sentence, could you say something like "It has also been tested with rust and is expected to work with..."

Reply:
I made the change WRT rust, but I now use weaker language for swift et al.